link_shortener/view_routes.py

```diff
@@ -68,11 +69,16 @@ async def about_page(request):
 @login_required
 @credential_whitelist_check
 async def all_active_links(request, user):
-    link_data = await retrieve_links(request, filters={'is_active': True})
+    filters = get_filter_dict(request)
+
+    link_data = await retrieve_links(request,
+                                     {'is_active': filters["is_active"]})
```

Comment:
Přežiju ledacos ale toto asi bohužel ne 😄 
Změň to prosím na
```
link_data = await retrieve_links(
    request,
    {'is_active': filters['is_active']}
)
```

Jednak mícháš různý uvozovky na jednom řádku, a druhak z toho indentu bych dostal psotník 😄

Reply:
fixnute
515d349b1310c996b6102f17967d11bbf489074f